plugin-template

```diff
@@ -113,6 +113,7 @@ DEPRECATED_FILES = {
         ".travis",
         ".travis.yml",
         "flake8.cfg",
+        "CHANGES/.TEMPLATE.rst",
```

Comment:
Maybe we should not do this after all. Sorry about that.

Reply:
Why not?

Reply:
You are right. My brain tricked me and thought about CHANGES.rst. That one we should delete explicitly after translating.